docs/threshold_encryption.md

```diff
@@ -0,0 +1,36 @@
+# Threshold Encryption
+*Threshold encryption* is one of cryptosystem, if in order to decrypt an encrypted message or to sign a message, several 
+parties (more than some threshold number) must cooperate in the decryption or signature protocol.
+The message is encrypted using a `public key` and the corresponding `private key` is shared among the participating parties.
+Only by possessing more than a threshold number of bits of the secret can the secret be determined.
```

Comment:
If I understand this line correctly.., In my opinion, `~the secret, we can determine the secret` looks more smooth. Or how about represent like `The secret can be determined only by possessing ~` form?